### ISSUE TITLE
Dutch locl feature

### ISSUE THREAD

**Issue by moyogo** [googlefonts/googlesans#563]
The locl feature for Dutch currently breaks Dutch texts by adding an acute to j when it is after an i with acute.

TLDR; The text of the spelling rules cannot be displayed in a meaninful way with the current locl feature. The spelling rules say the second accent on the ij is usually dropped and gives an example. This feature breaks text following the official rule "usually dropped"-part and breaks text written with spelling rules from before 1996.

The aim of the feature is to support the rule 5.1 of the official spelling: https://woordenlijst.org/leidraad/5/1 or as published in law https://wetten.overheid.nl/BWBR0019422/2006-05-01#Bijlage_Divisie5_Divisie5.1
> 5.1 Klemtoonteken
> regel 5.A
> Het klemtoonteken is ´. Als een klinker of tweeklank met twee of meer letters geschreven wordt, krijgen de eerste twee een klemtoonteken.
> *dat is dé oplossing*
> *het kán*
> *vóórkomen*
> *voorkómen*
> *je moet het vóór je verjaardag doen*
> *dóén*
> *níéuwe*
> Door technische beperkingen vervalt meestal het nadrukteken op de j van een lange *ij*. Bijvoorbeeld: blíjven kijken!

This can be translated as:
> 5.1 Stress marks
> rule 5.A
> The stress mark is ´. If a vowel or diphthong is written with two or more letters, the first two get a stress mark.
> [...]
> Due to technical limitations, the stress mark on the j of a lange *ij* is usually omitted. For example: *blíjven kijken!*

The fonts use the locl feature to add an acute to j when it is after an i with acute:
https://github.com/googlefonts/googlesans/blob/35577ff11a979c7c0d772d7cec75f3ae9cb00f43/source/GoogleSans/family.fea#L215-L224
This feature and its variations were popularized with the Glyphs App [Localize your font: accented Dutch IJ](https://glyphsapp.com/learn/localize-your-font-accented-dutch-ij) tutorial and the Underware blogpost [Typesetting Dutch IJ](https://www.underware.nl/blog/2014/10/typesetting-the-dutch-ij/).

As discussed in https://github.com/googlefonts/googlesans/pull/517 this locl feature hack:
* breaks non-Dutch person and place names used in Dutch texts.
* breaks Dutch text written with pre-1996 spelling rules as the current rule wasn’t settled until the 1996 official rules and some typesetting guids used a rule where only the first letter of dipthong had a stress mark. Given the high frequency of other dipthongs with only one stress mark in current Dutch texts, about a 1/3 or 1/2 of Dutch users still follow pre-1996 rules.

Additionaly, the reason why this issue is opened here, this Dutch locl feature hack:
* breaks the text of the rule it aims to support.

Specifically, the part of the rule where an example for the usually omitted stress mark on lange *ij* cannot be displayed as published in law with fonts that do the Dutch locl feature hack :
> Due to technical limitations, the stress mark on the j of a lange *ij* is usually omitted. For example: *blíjven kijken!*

See also Rainer’s comment, in a private discussion, about the Glyphs App Dutch locl feature tutorial:
<img width="605" alt="Screenshot 2022-10-31 at 18 24 28" src="https://user-images.githubusercontent.com/1923747/206390645-4dad8e97-aa32-4a42-9979-0e69acc5f19a.png">

@MariannaPaszkowska @chrissimpkins @davelab6

**Comment by moyogo** [comment]
@MariannaPaszkowska @chrissimpkins bumping this as Dalton Maag’s latest fonts (Bankside Sans and Bankside Serif) don’t do the iacute-j feature.
